config.go

```diff
@@ -662,6 +708,11 @@ func DefaultConfig() Config {
 		Htlcswitch: &lncfg.Htlcswitch{
 			MailboxDeliveryTimeout: htlcswitch.DefaultMailboxDeliveryTimeout,
 		},
+		GRPC: &GRPCConfig{
+			ServerPingTime:    defaultGrpcServerPingTime,
+			ServerPingTimeout: defaultGrpcServerPingTimeout,
+			ClientPingMinWait: defaultGrpcClientPingMinWait,
+		},
```

Comment:
Why is there no defaultGrpcClientAllowPingWithoutStream variable defined here too?

Reply:
Because the default of a boolean variable is always `false`.

Reply:
I was wondering that but as a comparison `defaultKeepFailedPaymentAttempts` was defined above to be false, so I assumed it was necessary, but I guess `defaultKeepFailedPaymentAttempts` has a redundant default definition?